velox/docs/functions/presto/math.rst

```diff
@@ -255,6 +255,13 @@ Probability Functions
     The successProbability must be real value in [0, 1], numberOfTrials and value must be
     positive integers with numberOfTrials greater or equal to value
 
+.. function:: inverse_binomial_cdf(numberOfTrials, successProbability, p) -> integer
```

Comment:
In the Presto documentation these functions are separated into 2 lists "Probability functions : cdf" and "Probability functions : inverse_cdf". Lets follow the same in Velox docs also.